utils.py

```diff
@@ -29,6 +29,35 @@ def get_cbl_name():
     full_config = os.environ["CONFIG"]
     base_config = full_config.split("+")[0]
 
+    # Distribution configurations have a URL
+    if "https://" in base_config:
+        if "fedora" in base_config:
+            fedora_to_cbl = {
+                "aarch64": "arm64",
+                "armv7hl": "arm32_v7",
+                "i686": "x86",
+                "s390x": "s390"
+            }
+            # The URL is https://.../kernel-<arch>-fedora.config
+            fedora_arch = base_config.split("/")[-1].split("-")[1]
+            if fedora_arch in fedora_to_cbl:
+                return fedora_to_cbl[fedora_arch]
+            return fedora_arch
+        if "openSUSE" in base_config:
+            suse_to_cbl = {
+                "armv7hl": "arm32_v7",
+                "i386": "x86",
+                "s390x": "s390"
+            }
+            # The URL is https://.../<arch>/default
+            suse_arch = base_config.split("/")[-2]
+            if suse_arch in suse_to_cbl:
+                return suse_to_cbl[suse_arch]
+            return suse_arch
```

Comment:
perhaps just `return suse_to_cbl[suse_arch]` so that we fail loudly for unknown arches? Or are there values not in suse_to_cbl?

Reply:
`arm64`, `ppc64le`, and `x86_64` are left out because they are a one to one map. As above, I can add them and just `return suse_to_cbl[suse_arch]` if you prefer?